### PR TITLE
Add the Desktop-version to Portfolio

### PR DESCRIPTION
**Pull request by Mujeeb4582**
## Changes done
Modify the mobile-version 
Add the required images to the image folder.
Add the media query for the desktop version (Desktop: 768px or wider)
Add the required tag for the desktop version in the index.html file.
Add the styles for the required tag in the style.css file.

### Note:
Kindly, If you don't understand anything feels free to contact me and tag me in the comment section. I will answer you.
Happy coding 👏 👏 👏

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -277,76 +289,94 @@ <h3 class="Skills-title">Skills</h3>
...
+            class="first_name"
             type="text"
-            name="user_name"
-            placeholder="Full name"
+            name="first_name"
+            placeholder="First name"
             maxlength="30"
-            required
-          />
+            required />
```
Comment (leonardodiasb) [inline index.html]: -[ ] There should be one input for Full name (just like the mobile version) and this should take the space of both First Name and Last Name inputs (shown in the desktop version). Kindly remove the inputs for the first and last name and use only the full name input.

File: index.html [hunk 2]
```diff
@@ -277,76 +289,94 @@ <h3 class="Skills-title">Skills</h3>
...
+        >
+        <button class="dtouch" type="submit">Get in touch</button>
+      </div>
+    </form>
+
+    <!-- Form end -->
+
+    <!-- Social section start -->
+
+    <footer class="social-contact">
```
Comment (leonardodiasb) [inline index.html:330]: - [ ] There should be a gray line on top of the social media links. Kindly add that gray line to make your portfolio design similar to the Figma template.
 -Your design:
image

-Figma template:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Mujeeb4582, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Good UI design. ✔️
 - No linter errors. ✔️ 
 - Professional PR. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] Your desktop design looks great and is very similar to the Figma template. But, on desktop screens with sizes close to the transition from mobile to desktop(768px wide), there are some issues. The 'Works' section and technologies cards in the 'About' section are cut out of the screen due to the non-responsiveness behavior of these sections. Please make it responsive so all the content can fit on the screen in small desktop screen sizes.
- [ ] The form on small desktop screens also is very reduced to the fixed width of the text on its side. Please make it more responsive so the user can have a better experience. You could substitute the width to a `max-width` for example. 
Screen Recording 2022-09-29 at 1 00 40 PM (1)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Mujeeb4582) [reply index.html]: image


@leonardodiasb look at the desktop version of the form there is two input for the name. still i can change to just full name.

Comment (Mujeeb4582) [reply index.html]: @leonardodiasb please reply me??????

**Comment by Mujeeb4582** [comment]
# Changes required Done
- check the last three commits

**Review by Olamarx — APPROVED** [review]
Hi @Mujeeb4582, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Correct use of Git and Github flow ✔️ 
- Project works as required 👍🏼
- Descriptive Readme ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**